scrapi/base/__init__.py

```diff
@@ -211,12 +211,9 @@ def get_properties(self, result, property_list):
             prop.extend(result.xpath(
                 '//ns0:{}/node()'.format(item),
                 namespaces=self.NAMESPACES
-            ) or [''])
+            ))
 
-            if len(prop) > 1:
-                properties[item] = [self.copy_to_unicode(item) for item in prop]
-            else:
-                properties[item] = self.copy_to_unicode(prop[0])
+            properties[item] = [self.copy_to_unicode(element) for element in prop]
```

Comment:
Well, originally it was debugging for when the es mapping broke, but that wasn't the problem in the end. We decided to keep it, because we realized that trying to programmatically iterate through all of the properties, and them having inconsistent data types might be annoying (most especially when there is only one of an item in one record, and multiple in another)